site/docs/relations/physical_relations.md

```diff
@@ -17,14 +17,14 @@ The hash equijoin join operator will build a hash table out of the right input b
 
 ### Hash Equijoin Properties
 
-| Property            | Description                                                  | Required                 |
-| ------------------- | ------------------------------------------------------------ | ------------------------ |
-| Left Input          | A relational input.                                          | Required                 |
-| Right Input         | A relational input.                                          | Required                 |
-| Join Expression     | A boolean condition that describes whether each record from the left set "match" the record from the right set. The condition must only include the following operations: AND, ==, field references, is not distinct from. Field references correspond to the direct output order of the data. | Required.                |
+| Property            | Description                                                                                                                                                                                                             | Required                 |
+|---------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------|
+| Left Input          | A relational input.                                                                                                                                                                                                     | Required                 |
+| Right Input         | A relational input.                                                                                                                                                                                                     | Required                 |
```

Comment:
Either we should assign these as probe and build (my preference would be to be explicit) or we should specifically explain that consumers should assign probe & build based on whatever criteria they want.  As it is, I think this is ambiguous.

Reply:
I found that the documentation has mentioned that the right input is build-side, see [this](https://substrait.io/relations/physical_relations/#hash-equijoin-operator)

I also updated in this section [https://substrait.io/relations/physical_relations/#hash-equijoin-properties]